components/use-crossposter.js

```diff
@@ -53,12 +53,15 @@ export default function useCrossposter () {
   return useCallback(async item => {
     let failedRelays
     let allSuccessful = false
+    let eventId
```

Comment:
If we're calling it `noteId` elsewhere, then we should probably call it that here.